app/cli/internal/action/workflow_run_describe.go

```diff
@@ -49,6 +49,7 @@ type WorkflowRunItemFull struct {
 
 type WorkflowRunAttestationItem struct {
 	Envelope    *dsse.Envelope `json:"envelope"`
```

Comment:
is this property used anymore?

Reply:
Yes, it's still used as a fallback for old clients

Reply:
but you are the client no? I mean, this change will not apply to other clients?

Reply:
Actually, if we remove it, we won't be able to show the envelope for old servers. I'd keep it as a fallback for some time.